resources/cmake/ConnextDdsAddExamples.cmake

```diff
@@ -0,0 +1,84 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+
+#[[.rst
+.. connextdds_add_examples:
+
+ConnextDdsAddExamples
+---------------------
+
+Function to build a bunch of examples.
+
+::
+
+    connextdds_add_examples(
+        EXAMPLES example1 [...]
+        [SKIP condition]
+        [SKIP_MESSAGE message]
+    )
+
+Add a bunch of examples to be built. This method adds subdirectories that
+contains an example with a CMakeList.txt and generates the build system
+for all of them.
+
+``EXAMPLES`` (required):
+    List of examples names that can be compiled with the given MIN_VERSION
+    of RTIConnextDDS.
+``SKIP`` (optional):
+    If defined, the examples will be skipped.
+``SKIP_MESSAGE`` (optional):
+    If the examples are going to be skipped, prints this message after
+    ``Skipping examples because``.
+]]
+
+include(CMakeParseArguments)
```

Comment:
we should put the includes at the beginning of the file, and the documentation of a function right before the definition of that function

Reply:
All the others .cmake files have the includes after the documentation and the function before the includes.